substrate/frame/assets/src/types.rs

```diff
@@ -322,4 +322,10 @@ where
 pub struct AssetParameters {
 	/// Is eresidency required to receive the asset
 	pub(super) eresidency_required: bool,
-}
\ No newline at end of file
+}
+
+#[derive(Copy, Clone, Encode, Decode, Eq, PartialEq, Default, RuntimeDebug, MaxEncodedLen, TypeInfo)]
+pub struct AssetRelatedCompany {
+	/// Company related to the asset
+	pub(super) related_company_id: u32,
+}
```

Comment:
Any reason to wrap single value in a struct? Maybe just:
```
type AssetRelatedCompany = u32;
```
would suffice?

Reply:
pub type